portfolio/src/components/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import clsx from "clsx";
+import type { ButtonHTMLAttributes, FC } from "react";
+
+type StyleVariant = "primary" | "secondary";
```

Comment:
I noticed that you only had two different types of buttons on the page: a primary button that was colored and a secondary button that wasn't colored and just had a white border. Using this type (also known as a string literal type), we can force any `Button` instance to have a set style variant, either `primary` or `secondary`. And then using this prop that we pass into the component, we can apply specific styling to `primary` buttons and specific styling to `secondary` buttons